src/app/atoms/action-modal/action-modal.tsx

```diff
@@ -12,16 +18,29 @@ interface ActionModalProps {
   className?: string;
 }
 
-export const ActionModal = memo<ActionModalProps>(({ onClose, children, hasCloseButton = true, title }) => (
-  <CustomModal isOpen overlayClassName="backdrop-blur-xs" onRequestClose={onClose}>
-    <div className="relative p-3 border-b-0.5 border-lines w-modal">
-      <h1 className="text-center text-font-regular-bold mx-9">{title}</h1>
-      {hasCloseButton && (
-        <Button className="absolute top-3 right-3" onClick={onClose}>
-          <IconBase Icon={CloseIcon} size={16} className="text-grey-2" />
-        </Button>
+export const ActionModal = memo<ActionModalProps>(({ onClose, children, hasCloseButton = true, title }) => {
+  const { fullPage } = useAppEnv();
+
+  return (
+    <CustomModal
+      isOpen
+      className="rounded-lg"
+      overlayClassName={clsx(
+        'backdrop-blur-xs',
+        fullPage && actionModalStyles.fullPageOverlay,
+        fullPage && `top-9 bottom-8 min-h-80 rounded-md ${LAYOUT_CONTAINER_CLASSNAME}`
```

Comment:
Sorry, this one shouldn't, see bug 2 here: https://madfish.atlassian.net/browse/TW-1555?focusedCommentId=15273

Reply:
Unfortunately, placing a portal of `react-modal` won't help a lot because an overlay has to have `fixed` position if possible to prevent the covered content from scrolling. However, I'll add some changes to use common values